dataflux_pytorch/tests/test_dataflux_lightning_checkpoint.py

```diff
@@ -74,7 +98,18 @@ def test_valid_path(self):
             except:
                 self.fail(msg=f'Valid path: {p} parsed but failed')
 
-    def test_save_and_load_checkpoint(self):
+    def test_valid_object_path(self):
+        paths = {
+            Path(f'gs://{self.bucket_name}/path'),
+            Path(f'gcs://{self.bucket_name}/path')
```

Comment:
Were you able to confirm that creating the Path object like this causes str(path) to start with "gs:/" or "gcs:/" instead of "gs://" and "gcs://"? If not, would it make sense to add paths to this test that are constructed directly with "gs:/" and "gcs:/" also?

Reply:
Yes because of how Path handles urls,
All the following examples when converted to string would yield `gcs:/bucket-name`
`gcs:/bucket-name`
`gcs:///bucket-name`
`gcs://bucket-name`